<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSurfaceDialog</class>
 <widget class="QDialog" name="AddSurfaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Surface</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxObjectType">
      <item>
       <property name="text">
        <string>Rectangle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Circle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangle</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="openingCheckBox">
      <property name="text">
       <string>Opening</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
